### PR TITLE
Apple SDK Feb release - Version 1.14.0

### PR DESCRIPTION
**Pull request by UdayaNalla**
## Description

Apple SDK Feb release - Version 1.14.0 

Updated the below files
1.  Version updated in CHANGELOG.md
2. Version updated in all 12 jazzy.yaml files
3. Version updated in config/version.xcconfig
4. Version updated in AppCenter.podspec

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # App Center SDK for iOS and macOS Change Log
 
-## Version 1.13.1 (Not yet released)
+## Version 1.14.0
```
Comment (guperrot) [inline CHANGELOG.md]: please use 1.13.1 as its bug fix only.

Comment (ElektrojungeMS) [reply CHANGELOG.md]: What Guillaume says. Post-Release of 1.13.1, the next version will be 1.13.2.

File: Documentation/iOS/AppCenterAnalytics/.jazzy.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ sdk: iphonesimulator
 theme: ../../Themes/apple
 
 module: AppCenterAnalytics
-module_version: 1.13.1
+module_version: 1.14.0
```
Comment (ElektrojungeMS) [inline Documentation/iOS/AppCenterAnalytics/.jazzy.yaml]: Please also fix this version change. Thx!

Comment (UdayaNalla) [reply Documentation/iOS/AppCenterAnalytics/.jazzy.yaml]: All version changes have been reverted. Version is now set as 1.13.1 in all files.

**Pull request merged**